data/distro.yml

```diff
@@ -1,790 +1,612 @@
 - name: "Ubuntu"
   logo: "ubuntu.svg"
-  info: >
-    <ol class="distrotut">
-      <li>
-        <h2>Install Flatpak</h2>
-        <p>To install Flatpak on Ubuntu 18.10 (Cosmic Cuttlefish) or later, simply run:</p>
-        <terminal-command>sudo apt install flatpak</terminal-command>
-        <p>With older Ubuntu versions, the official Flatpak PPA is the recommended way to install Flatpak. To install it, run the following in a terminal:</p>
-        <terminal-command>
-          sudo add-apt-repository ppa:flatpak/stable
-          sudo apt update
-          sudo apt install flatpak
-        </terminal-command>
-      </li>
-      <li>
-        <h2>Install the Software Flatpak plugin</h2>
-        <p>The Flatpak plugin for the Software app makes it possible to install apps without needing the command line. To install, run:</p>
-        <terminal-command>sudo apt install gnome-software-plugin-flatpak</terminal-command>
-        <p>Note: the Software app is distributed as a Snap since Ubuntu 20.04 and does not support graphical installation of Flatpak apps. Installing the Flatpak plugin will also install a deb version of Software and result in two Software apps being installed at the same time.</p>
-      </li>
-      <li>
-        <h2>Add the Flathub repository</h2>
-        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
-        <terminal-command>
-          flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
-        </terminal-command>
-      </li>
-      <li>
-        <h2>Restart</h2>
-        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
-      </li>
-    </ol>
+  steps:
+    - name: Install Flatpak
```

Comment:
Another small thing. Sometimes, there are no quotation marks or there are either single quotation marks or double ones. It would be great to make this more coherent.

Reply:
We can do that for the name, but in other cases where we have html inside it might be a bad idea

Reply:
You're right.

Reply:
Changed, but I haven't runtime tested it

Reply:
FWIW it's parsed fine (just tested)